src/main/java/com/jcabi/github/RtRepo.java

```diff
@@ -98,6 +105,22 @@
             .path(this.coords.repo())
             .back();
     }
+    
+    public Iterable<String> invitees() throws IOException {
+    	Iterator<JsonValue> iter = this.request.uri().path("/invitations").back().method(Request.GET)
+		.body().set(Json.createArrayBuilder().build()).back()
```

Comment:
@ShalomGottesman Why do you specify an empty Json body? I am pretty sure it is not needed :)